tippilab/tippilab.com
# Create contacts page

**How to test**

1) go to the /contact/ page
2) send the message
3) if everything is ok  -  you must be redirect to the /success/ page
4) check your console 

TODO:

- [x] -integrate with Sendgrid using django-sendgrid.
- [x] -create free account in sendgrid
